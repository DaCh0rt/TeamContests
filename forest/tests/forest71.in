146 134
-1579 -5375
-9150 1217
-8148 2936
-8013 8685
3197 3964
-7474 -4046
-114 4880
-797 -6428
-5116 2691
6108 -5502
-1020 -510
-5239 9668
-2338 -3090
-3709 5753
-6291 -3116
-4620 -2946
3625 -1178
1664 -2411
-3243 50
-854 -4012
1208 -3370
-751 -2470
5228 -6056
1367 8062
3195 5784
-6464 -8166
4201 8428
4137 -567
-1842 2763
-3561 -8723
5980 -6417
8830 2477
2478 8786
6145 -2609
1842 -6784
-6793 -2582
-7240 6184
7465 5796
353 7755
5102 5253
-6422 3255
-7153 -3536
9793 -8975
9343 -3166
-1176 949
-4531 -8044
-3110 1279
6692 654
9039 -6743
186 -6270
7149 -4817
-2250 9608
2508 4439
-2851 9926
-8594 -8902
-7967 -1342
9622 -4407
707 8652
-40 1827
6635 -5017
-6612 5042
255 -3334
-7890 -5099
-5086 5613
6871 -4474
4477 6401
9661 -737
-9621 6608
-3462 -7893
-4587 6038
1813 -1090
-878 2694
-869 -7435
-2604 -4084
2583 -8675
4263 -6370
-2625 -3062
691 -1992
2425 -3350
3605 9553
-6835 3867
7659 9959
-5145 8185
6303 -5307
3688 3834
551 4621
-4252 4940
2907 385
-3508 7558
-7261 -3829
-2185 1740
-1902 -8547
5806 460
-9276 9579
4074 9470
-6796 6695
-4649 3802
4886 -1467
3786 -5810
-1126 -2815
5973 7504
-6689 3225
-376 -8997
2028 1143
693 5078
8278 1656
-3540 -1110
2758 743
-6675 1853
6646 -2373
7087 -5388
-3622 -5015
-7266 9916
912 -2373
477 -3709
3522 7434
6948 -969
-1444 7588
802 -8865
337 -7264
4444 -3530
-8602 183
4970 -7527
7118 2318
-3930 -8666
4994 -7369
4113 7957
3696 3694
-4220 -1249
5170 -3739
838 8107
2882 -7203
-1672 -9162
3513 -7236
-4011 2345
4925 -6935
-6190 -8696
-5490 63
-3299 -2578
7874 -5866
-5912 4766
8211 3978
-2737 -7864
-901 -4645
9178 -9114
3531 -4499
