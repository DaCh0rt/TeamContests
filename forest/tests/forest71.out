73868.30 357541231.85
